customization/XFCE.md

```diff
@@ -39,7 +35,6 @@ Things needs/recommended to be done:
 -   create own favorites menu (currently standard XFCE menu isn't modified to use per-VM subsections, which makes it very inconvenient):
     1.  create \~/.config/menus/favorites.menu, example content:
 
-        ```
         <!DOCTYPE Menu PUBLIC "-//freedesktop//DTD Menu 1.0//EN"
```

Comment:
~~~ html
    code comes here
    ~~~

Works nicely. The only difference is that the `~~~html` syntax adds line numbers …
Currently not supported with